apps/editing-toolkit/editing-toolkit-plugin/dotcom-fse/helpers.php

```diff
@@ -131,9 +131,32 @@ function is_theme_supported() {
 		// get_theme_slug will always return false.
 		return false;
 	}
+	$slug = get_theme_slug();
 	// Use un-normalized theme slug because get_theme requires the full string.
-	$theme = wp_get_theme( get_theme_slug() );
-	return ! $theme->errors() && in_array( 'full-site-editing', $theme->tags, true );
+	$theme      = wp_get_theme( $slug );
+	$theme_slug = normalize_theme_slug( $slug );
+	return ! $theme->errors() && in_array( $theme_slug, get_supported_themes(), true );
```

Comment:
Do we even need `$theme` / `wp_get_theme` anymore?

Reply:
We probably don't, but we were checking for errors before and so we'll keep it.